Chargebee/Classes/Cache/CBCache.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CBCache.swift
+//  Chargebee
+//
+//Created by ramesh_g on 28/07/23.
+//
+
+import Foundation
+
+
+struct ConfigCacheModel: Codable {
```

Comment:
Can we make all these `private` or `internal` as appropriate?

Reply:
> Can we make all these `private` or `internal` as appropriate?

Sure updated